greenriver/hmis-frontend
# Support form duplication for managedInVersionControl forms

## Description

Issue: https://github.com/open-path/Green-River/issues/6763
Backend PR: https://github.com/greenriver/hmis-warehouse/pull/4869

* Hide "edit form" button for forms that are managed in version control
* Add button for new "Duplicate Form" action
* Add tag showing if a form is managed in version control (here in the UI I'm still calling it "System Form" - open to suggestions)


## Type of change

## Testing Instructions
* Run `rails driver:hmis:seed_definitions` to set the flag (tested with different clients) and checkout backend
* In config tool you should see:
  * system-managed forms are flagged and sorted to the end
  * when you open a system-managed form, there is an alert saying contact support to edit, and there is a Duplicate Form button
  * the Duplicate button works and creates a new form
  * if there were custom fields in the base form, the new form collects onto a new cded (not the same one) after publish. I tested this by temporarily turning on the "duplicate" button for a non-system form that I knew had CDEs on it.


## Checklist before requesting review
